manifest_staging/charts/csi-secrets-store-provider-azure/templates/arc-monitoring.yaml

```diff
@@ -0,0 +1,213 @@
+{{- if .Values.enableArcExtension }}
+{{- if .Values.arc.enableMonitoring }}
+{{- if .Values.linux.enabled}}
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: arc-{{ template "sscdpa.fullname" . }}-monitoring
+  namespace: {{ .Release.Namespace }}
+{{ include "sscdpa.arc.labels" . | indent 2 }}
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: "arc-{{ template "sscdpa.fullname" . }}"
```

Comment:
ditto (Maybe use static values for these?)

Reply:
updated